gobblin-core/src/main/avro/StandardManifestRecord.avsc

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
side question: do you plan also open source or writeup design around this. it would be quite useful for the community

Reply:
Yes. I'll create the initial documentation as a separate pr. Once Lei(@autumnust) open sourced the salesforce bulk writer, it would set up a good example about how to use the `StandardManifestRecord` and complete the documentation.